cpp/include/cugraph_c/array.h

```diff
@@ -55,44 +63,87 @@ cugraph_error_code_t cugraph_type_erased_device_array_create(
  */
 void cugraph_type_erased_device_array_free(cugraph_type_erased_device_array_t* p);
 
+#if 0
```

Comment:
Better delete if no longer necessary?

Reply:
Added a FIXME in the latest push.  This was going to be part of the API, but it can't currently be implemented.  We need to have a discussion about whether we can implement this or whether we should address this a different way.